python/dgl/graphbolt/negative_sampler.py

```diff
@@ -41,7 +42,7 @@ def _sample(self, node_pairs):
         """
         Generate a mix of positive and negative samples.
 
-        Parameters
+        node_pair
         ----------
         node_pairs : Tuple[Tensor] or Dict[etype, Tuple[Tensor]]
```

Comment:
Should the input of the _sample also be LinkPredictionBlock?

Reply:
Yes, this will be changed after mini-batch sampler change is completed.